plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/test-account/index.tsx

```diff
@@ -0,0 +1,298 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useRef, useEffect } from 'react';
+import apiFetch from '@wordpress/api-fetch';
+import { Loader } from '@woocommerce/onboarding';
+import { __ } from '@wordpress/i18n';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Notice, Button } from '@wordpress/components';
+import { Link } from '@woocommerce/components';
+import { navigateTo, getNewPath } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import WooPaymentsStepHeader from '../../components/header';
+import { useOnboardingContext } from '../../data/onboarding-context';
+import { WC_ASSET_URL } from '~/utils/admin-settings';
+import './style.scss';
+
+interface StepCheckResponse {
+	status: string;
+	success: boolean;
+}
+
+const TestDriveLoader: React.FunctionComponent< {
+	progress: number;
+} > = ( { progress } ) => (
+	<Loader className="woocommerce-payments-test-account-step__preloader">
+		<Loader.Layout className="woocommerce-payments-test-account-step__preloader-layout">
+			<Loader.Illustration>
+				<img
+					src={ `${ WC_ASSET_URL }images/onboarding/test-account-setup.svg` }
```

Comment:
is it compressed?

Reply:
I took it directly from the WooPayments codebase, so I think it should be compressed 🤔